GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -508,6 +534,29 @@ void FramebufferManagerCommon::BlitFramebufferDepth(VirtualFramebuffer *src, Vir
 	dst->last_frame_depth_updated = gpuStats.numFlips;
 }
 
+TrackedDepthBuffer *FramebufferManagerCommon::GetOrCreateTrackedDepthBuffer(VirtualFramebuffer *vfb) {
+	for (auto tracked : trackedDepthBuffers_) {
+		// Disable tracking if color of the new vfb is clashing with tracked depth.
+		if (vfb->fb_address == tracked->z_address) {
+			tracked->vfb = nullptr;  // this is checked for. Cheaper than deleting.
+			continue;
+		}
+
+		if (vfb->z_address == tracked->z_address && vfb->z_stride == tracked->z_stride) {
```

Comment:
Hm... I wonder if it's better to wipe it if the z_stride doesn't match, or at least log.

I'm imagining a game that uses bloom, and thus changes the stride for smaller framebuffers but reuses the z.  This to me is a clear signal the z doesn't matter, and it'd be better to skip copying the depth if it returns to a new color buffer with the same z, than to copy the old depth from some now-ancient vfb.

That said, maybe there are other cases... possibly a good idea to log, but we probably build up outdated ones here.

Also, do we update this stride if the framebuf itself is updated?  In `DoSetRenderFrameBuffer()` we may change it, and a vfb could go on updating its depth.  At least that should track a new depth entry if it changes.

-[Unknown]

Reply:
Yes you're right, we should definitely zap here if the stride doesn't match but the address does. Adding that.

I'll rely on the mismatch to track a new depth entry if it changes I think instead of doing anything explicit, until we find a case where that doesn't work..